bitcoin/contrib/test.sh

```diff
@@ -81,7 +81,7 @@ cargo run --example taproot-psbt --features=bitcoinconsensus
 
 # Build the docs if told to (this only works with the nightly toolchain)
 if [ "$DO_DOCS" = true ]; then
-    RUSTDOCFLAGS="--cfg docsrs" cargo +nightly rustdoc --features="$FEATURES" -- -D rustdoc::broken-intra-doc-links
+    RUSTDOCFLAGS="--cfg docsrs" cargo +nightly rustdoc --all-features -- -D rustdoc::broken-intra-doc-links -D warnings || exit 1
```

Comment:
We have `set -ex` so this "fix" did not add any new behaviour to the script :(

Reply:
I think `-D` doesn't work for some reason.